gfosd/components/aggregate.py

```diff
@@ -37,7 +37,10 @@ def prepare_attributes(self, T, p=1):
         self._Pz = sp.block_diag([
             c._Pz for c in self._gf_list
         ])
-        self._make_q()
+        # same logic as above but for q
```

Comment:
add comment: this is related to "close to input vector" class, which is incomplete and ready for use. q is still set to be zero for all implemented classes.